zkstack_cli/crates/zkstack/build.rs

```diff
@@ -1,12 +1,22 @@
 use std::path::PathBuf;
 
+use anyhow::{anyhow, Context};
+use dirs::{config_local_dir, home_dir};
 use ethers::contract::Abigen;
 
-fn main() -> eyre::Result<()> {
+fn main() -> anyhow::Result<()> {
     let outdir = PathBuf::from(std::env::var("OUT_DIR")?).canonicalize()?;
-    Abigen::new("ConsensusRegistry", "abi/ConsensusRegistry.json")?
-        .generate()?
-        .write_to_file(outdir.join("consensus_registry_abi.rs"))?;
+    Abigen::new("ConsensusRegistry", "abi/ConsensusRegistry.json")
+        .map_err(|_| anyhow!("Failed ABI deserialization"))?
+        .generate()
+        .map_err(|_| anyhow!("Failed ABI generation"))?
+        .write_to_file(outdir.join("consensus_registry_abi.rs"))
+        .context("Failed to write ABI to file")?;
+
+    // Copy completion scripts
+    if copy_completion_scripts().is_err() {
+        println!("WARNING: It was not possible to install autocomplete scripts. Please generate them manually with `zkstack autocomplete`")
```

Comment:
could we print an error as well?

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3097/commits/ac22d84fe583dafad7674edec9da784a8fee6364